walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/socials.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Socials</title>
+
+
+  <style>
+    ul{
+    	list-style: none;
+    }
+    li{
+    	display: inline;
+    	float: left;
+    }
+    body{
+    	background-image: url("images/eggs.jpg");
+        background-size:100% 100vh;
+        font-famile: googleFont;
+    }
+    @font-face{
+        font-family: googleFont;
+        src: url("/fonts/Product Sans Regular.ttf");
+    }
+    h1{
+        font-size: 400%;
+        text-shadow: 2px 2px #626260;
+        text-align: center;
+        margin-top: 4vh;
+        padding: 2vh;
+    }
+    img{
+        width: 30vh;
+        height: 30vh;
+        margin: 2vh;
+        margin-top: 5vh;
+        transition: transform 1s;
+    }
+    img:hover{
+        transform: scaleX(-1);
+
+    }
+    #container{
+        height: 60vh;
+        width: 150vh;
+        background-color: #F6F4E9;
+        margin: auto;
+        opacity: 0.98;
+        border-radius: 50px;
+    }
+      </style>
+
+</head>
+
+<body>
+  
+<div id="container">
+<h1>Connect On My Socials</h1>
```

Comment:
nit: format, 2 spaces extra for children elements

Reply:
Done!